dash-renderer/MANIFEST.in

```diff
@@ -1,7 +1,4 @@
-include dash_renderer/dash_renderer.dev.js
-include dash_renderer/dash_renderer.dev.js.map
-include dash_renderer/dash_renderer.min.js
-include dash_renderer/dash_renderer.min.js.map
-include dash_renderer/react-dom@*.min.js
-include dash_renderer/react@*.min.js
-include dash_renderer/prop-types@*.js
+include package.json
+include digest.json
+include dash_renderer/*.json
```

Comment:
```suggestion
include dash_renderer/*.js
```
?

Reply:
🙈  typical typo when you are about to sleep, try some last shots with CI.